lua/telescope/script.lua

```diff
@@ -0,0 +1,130 @@
+local Job = require'plenary.job'
+
+local download_file = function(source, callback)
```

Comment:
Didn't check the size of the files, but would be nice to have a caching option to avoid unnecessary network requests

Reply:
We would have to download the file upfront with `lua require'telescope.script'.get_emoji_source()` download a file, modify it and then save the file as `.local/share/nvim/telescope-sources/emoji.txt`.
I agree downloading the file every time when running the picker is "stupid", that is just a workaround to not provide the files in the repo and a easy way to do version bumps.

But now that we will end up with an extension i think i will keep the script and do a github action that will create these files and push them to the repo. Maybe all maybe just some. I will ping you when i created the new PR for the extensions. I am currently porting